src/Rezzza/AliceExtension/Doctrine/ORMPersister.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Rezzza\AliceExtension\Doctrine;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Nelmio\Alice\ORM\Doctrine;
+
+class ORMPersister extends Doctrine
```

Comment:
In fact we should implements `Nelmio\Alice\ORMInterface` instead as we override the complete original behavior ?